Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndDigitisation.py

```diff
@@ -0,0 +1,256 @@
+extHcalReadoutName = "ExtBarHCal_Readout"
+newExtHcalReadoutName = extHcalReadoutName + "_phieta"
+
+from Configurables import SimG4Alg, SimG4SaveCalHits
+saveecalbarreltool = SimG4SaveCalHits("saveECalBarrelHits", readoutNames = [ecalBarrelReadoutName])
+saveecalbarreltool.positionedCaloHits.Path = "ECalBarrelPositionedHits"
+saveecalbarreltool.caloHits.Path = "ECalBarrelHits"
+savecalendcaptool = SimG4SaveCalHits("saveCalEndcapHits", readoutNames = [calEndcapReadoutName])
+savecalendcaptool.positionedCaloHits.Path = "CalEndcapPositionedHits"
+savecalendcaptool.caloHits.Path = "CalEndcapHits"
+savecalfwdtool = SimG4SaveCalHits("saveCalFwdHits", readoutNames = [calFwdReadoutName])
+savecalfwdtool.positionedCaloHits.Path = "CalFwdPositionedHits"
+savecalfwdtool.caloHits.Path = "CalFwdHits"
+savehcaltool = SimG4SaveCalHits("saveHCalHits",readoutNames = [hcalReadoutName])
+savehcaltool.positionedCaloHits.Path = "HCalPositionedHits"
+savehcaltool.caloHits.Path = "HCalHits"
+saveexthcaltool = SimG4SaveCalHits("saveExtHCalHits",readoutNames = [extHcalReadoutName])
+saveexthcaltool.positionedCaloHits.Path = "ExtHCalPositionedHits"
+saveexthcaltool.caloHits.Path = "ExtHCalHits"
+
+
+# next, create the G4 algorithm, giving the list of names of tools ("XX/YY")
+from Configurables import SimG4SingleParticleGeneratorTool
+pgun = SimG4SingleParticleGeneratorTool("SimG4SingleParticleGeneratorTool",saveEdm=True,
+                particleName=particleType,energyMin=energy,energyMax=energy,etaMin=-5.0,etaMax=5.0,
+                OutputLevel = INFO)
+
+geantsim = SimG4Alg("SimG4Alg",
+                       outputs= ["SimG4SaveCalHits/saveECalBarrelHits", "SimG4SaveCalHits/saveCalEndcapHits", "SimG4SaveCalHits/saveCalFwdHits", "SimG4SaveCalHits/saveHCalHits", "SimG4SaveCalHits/saveExtHCalHits"],
+                       eventProvider=pgun,
+                       OutputLevel=INFO)
+
+#Configure tools for calo reconstruction
+# EM scale calibration
+from Configurables import CalibrateCaloHitsTool
+calibHcells = CalibrateCaloHitsTool("CalibrateHCal", invSamplingFraction="34.5")
+
+from Configurables import CalibrateInLayersTool
+calibEcalBarrel = CalibrateInLayersTool("CalibrateECalBarrel",
+                                   # sampling fraction obtained using SamplingFractionInLayers from DetStudies package
+                                   samplingFraction = [0.12125] + [0.14283] + [0.16354] + [0.17662] + [0.18867] + [0.19890] + [0.20637] + [0.20802],
+                                   readoutName = ecalBarrelReadoutName,
+                                   layerFieldName = "layer")
+calibEndcap = CalibrateInLayersTool("CalibrateEndcap",
+                                    # sampling fraction obtained using SamplingFractionInLayers from DetStudies package
+                                    # HEC -> extrapolation from Ecal EC (0.15 * active / passive thickness)
+                                    samplingFraction = [0.15] * 118 + [0.03] * 78,
```

Comment:
The calibration is applied to the readout of the EMEC only, isn't it?
For the calibration of the HEC another calibration tool has do be used, hasn't it?

Reply:
The calibration is done using 2 different sampling fractions, one for ECAL part and the second one for the HCAL part. The calibration for HEC and forward hadronic calo was not calculated properly, it is a first guess. I extrapolated the numbers from the EMEC and EFCAL calibrations taking into account different thicknesses of the passive/active material.

Reply:
That I understood, but the readout is done for the EM and hadronic parts seperately, isn't it?